datumaro/plugins/market1501_format.py

```diff
@@ -88,10 +88,11 @@ def _load_items(self, rootdir):
                     image=image_path)
                 items[item_id] = item
 
-            attributes = item.attributes
-            attributes['query'] = subdir == Market1501Path.QUERY_DIR
-            attributes['person_id'] = pid
-            attributes['camera_id'] = camid
+            if pid != Market1501Path.UNKNOWN_ID:
+                attributes = item.attributes
+                attributes['query'] = subdir == Market1501Path.QUERY_DIR
+                attributes['person_id'] = pid
+                attributes['camera_id'] = camid
```

Comment:
Why this information is not needed, when id is unknown?

Reply:
I think that we shouldn't include this information. If we initially didn't have attributes in the dataset, we shouldn't add them.

Reply:
I think, it is impossible for us to know at this point, if these values were present or not. Also, how about the `test` subset?

The second question is why do we exclude all the attributes?

Reply:
For example, we cannot check 'query' like that, so we check only one and it decides for all.

> I think, it is impossible for us to know at this point, if these values were present or not. Also, how about the `test` subset?
What do you mean with `test` subset?

Reply:
I mean the `bounding_box_test` directory from https://www.kaggle.com/pengcw1/market-1501/data, which contains images having -1, but camera ids and sequence ids are meaningful.

Reply:
Then we think we have no attributes when `person_id` and `camera_id` are -1.